main.tf

```diff
@@ -0,0 +1,136 @@
+locals {
+  app_name_and_env = "${var.app_name}-${var.app_env}"
+  name_tag_suffix  = "${var.app_name}-${var.customer}-${var.app_environment}"
+}
+
+/*
+ * Create cloudwatch log group for app logs
+ */
+resource "aws_cloudwatch_log_group" "cw_b2_fsbackup" {
+  name              = local.app_name_and_env
+  retention_in_days = 14
+
+  tags = {
+    name = "cloudwatch_log_group-${local.name_tag_suffix}"
+  }
+}
+
+/*
+ * Create required passwords
+ */
+resource "random_id" "b2_repo_password" {
```

Comment:
Can this use `random_password` instead? https://registry.terraform.io/providers/hashicorp/random/3.1.0/docs/resources/password